samples/subsys/shell/fs/boards/rcar_h3ulcb_ca57.overlay

```diff
@@ -0,0 +1,8 @@
+/* SPDX-License-Identifier: Apache-2.0 */
```

Comment:
EPAM copyright?

Reply:
as far as I see, it isn't a common approach, but Zephyr project has dozens overlay files without copyright